understanding/20/contrast-enhanced.html

```diff
@@ -236,7 +236,7 @@ <h3>Inactive User Interface Components</h3>
 
          <p>User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements. An inactive user interface component is visible but not currently operable. An example would be a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed.</p>
          <figure id="figure-grey-button-and-text">
-            <img src="img/inactive-button.png" alt="Grey button with non-contrasting grey text." width="120">
+            {% include "../21/img/1.4.11-ntc-button-inactive.svg" %}
```

Comment:
Heya @kfranqueiro, I found references to one of the updated assets in 2 other Understanding docs from WCAG 2.0: Contrast Enhanced and Contrast Minimum.

I went ahead and updated those to point explicitly to the `../21/img` directory. Is that okay?

Reply:
Discussed on Backlog call on 26 Sep, yes, this is fine; the version subfolders are purely an artifact of how the files were first organized, and crossing subfolders is not an issue.